### ISSUE TITLE
Version 1.3.7 rolled back Arrow to old 0.15.6

### ISSUE THREAD

**Issue by jmichalicek** [Koed00/django-q#571]
When upgrading from django-1 1.3.6 to 1.3.7, the version of Arrow used was rolled back from 1.1.0 to 0.15.6.
I can see in the [pyyproject.toml](https://github.com/Koed00/django-q/blob/v1.3.7/pyproject.toml#L47) shows `arrow = "^0.15.6"` while the old [setup.py](https://github.com/Koed00/django-q/blob/v1.3.6/setup.py#L40) which no longer exists had `    install_requires=['django>=2.2', 'django-picklefield', 'blessed', 'arrow'],`.

I assume this rollback to such an old version was not intentional. `pip-compile` (from pip-tools) fails if something, including the `requirements.in` tries to force a newer version of arrow.

My requirements.txt, generated by pip-tools, from django-q 1.3.6 included:
```
arrow==1.1.0 \
            --hash=sha256:8cbe6a629b1c54ae11b52d6d9e70890089241958f63bc59467e277e34b7a5378 \
            --hash=sha256:b8fe13abf3517abab315e09350c903902d1447bd311afbc17547ba1cb3ff5bd8 
            # via django-q
django-q[sentry]==1.3.6 \
            --hash=sha256:61ff587f268324b364c9aaeee92432e2c381d4480481ce8fc18a390ad357014a \
            --hash=sha256:901213e5515c6fbea63d2c57ee9e48f1012df1e958c515d2ba206118d829a3bd
            # via -r requirements.in
```

Whereas now:
```
arrow==0.15.8 \
    --hash=sha256:271b8e05174d48e50324ed0dc5d74796c839c7e579a4f21cf1a7394665f9e94f \
    --hash=sha256:edc31dc051db12c95da9bac0271cd1027b8e36912daf6d4580af53b23e62721a
    # via django-q
django-q[sentry]==1.3.7 \
    --hash=sha256:a3d38e1c0aad2c01e8a4f9d3890a2408fd1c0c8ee7c9263cc833f92b71b59e04 \
    --hash=sha256:bd2a2e2191af60c1a9af542768b15d4fb5e1546e46b893d2c7714f8136771c6e
    # via -r requirements.in
```

**Comment by Koed00** [comment]
Thanks for spotting that - I have no idea how or why arrow got pinned like that. Looks like I pinned that in the initial release over a year ago. Since I kept using pip-tools , it was never noticed.
Updating it now and I'll ready a new release.

**Comment by Koed00** [comment]
Just released 1.3.8 - hope that fixes it.

**Comment by jmichalicek** [comment]
Awesome, looks good to me and everything seems to be working, thank you!
